.github/workflows/dotnet.yml

```
@@ -142,6 +179,13 @@ jobs:
       API_IMAGE_TAG: ${{ fromJson(steps.meta.outputs.json).tags[0] }}
 
     steps:
+      - name: Extract version
+        id: sanitize-version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)